src/pdn/doc/define_pdn_grid.md

```diff
@@ -0,0 +1,80 @@
+## define_pdn_grid
+
+### Synopsis
+For specifying a power grid over the stdcell area
+```
+  % define_pdn_grid
+    [-name <name>] \
+    [-pins <list_of_pin_layers>] \
+    [-starts_with (POWER|GROUND)] \
+    [-voltage_domain <list_of_domain_names>] \
+    [-starts_with (POWER|GROUND)]
+```
+For specifying a power grid over macros in the design
+```
+  % define_pdn_grid
+    -macro \
+    [-name name] \
+    [-orient <list_of_valid_orientations>] \
+    [-instances <list_of_instances] \
+    [-cells <list_of_cells>] \
+    [-halo <list_of_halo_values>] \
+    [-voltage_domain <list_of_domain_names>] \
+    [-starts_with (POWER|GROUND)]    
```

Comment:
pin_direction is missing

Reply:
Updated.

Reply:
should this be: horizontalvertical -> horizontal|vertical ?